base/changes.txt

```diff
@@ -17,8 +17,7 @@ not part of the distribution.
 	* fontdef.dtx:
 	preload ts1lmr.fd in Unicode engines
 
-2024-06-19  Joseph Wright  <Joseph.Wright@latex-project.org>
-
```

Comment:
it has no been reverted, which isn't a problem either way, but I agree with @muzimuzhi 's approch @josephwright to clean up such things on the fly rather than making unnecessary pull requests for such a minor cleanup. I usually do that too when I notice one of those.

Reply:
can't type as usual. I meant: "If it has been reverted, so be it, but I agree with ...